mkosi/__init__.py

```diff
@@ -1259,14 +1259,32 @@ def gzip_binary(context: Context) -> str:
     return "pigz" if context.config.find_binary("pigz") else "gzip"
 
 
+def _kernel_get_ver(context: Context) -> str:
```

Comment:
That name is a bit generic, how about
```suggestion
def _kernel_get_ver_from_modules(context: Context) -> str:
```